CONTRIBUTING.md

```diff
@@ -1,26 +1,27 @@
-
-- [Git Know How / Reading List](#git-know-how--reading-list)
-- [PR Checklist - Contributor/Developer](#pr-checklist---contributordeveloper)
-- [PR Checklist - Maintainer/Reviewer](#pr-checklist---maintainerreviewer)
-- [Create local branch](#create-local-branch)
-  - [Directory structure](#directory-structure)
-- [Test Automation](#test-automation)
-  - [Summary](#summary)
-  - [Test Case Core Principles](#test-case-core-principles)
-- [Working with the Code](#working-with-the-code)
-  - [Running/Debugging the tests](#runningdebugging-the-tests)
-    - [Running a single test case](#running-a-single-test-case)
-    - [Running all test cases (unit+integration)](#running-all-test-cases-unitintegration)
-    - [Running unit tests only](#running-unit-tests-only)
-    - [Running integration tests only](#running-integration-tests-only)
-    - [What is npx used for?](#what-is-npx-used-for)
-    - [Debugging a test case](#debugging-a-test-case)
-  - [All-In-One Docker Images for Ledger Connector Plugins](#all-in-one-docker-images-for-ledger-connector-plugins)
-    - [Test Automation of Ledger Plugins](#test-automation-of-ledger-plugins)
-  - [Building the SDK](#building-the-sdk)
-  - [Adding a new public npm dependency to one of the packages:](#adding-a-new-public-npm-dependency-to-one-of-the-packages)
-  - [Adding a sibling package npm dependency to one of the packages:](#adding-a-sibling-package-npm-dependency-to-one-of-the-packages)
-  - [On Reproducible Builds](#on-reproducible-builds)
+# Contributing
```

Comment:
@AzaharaC Was this change produced by the auto-formatter or was it something needed to make the rendering work with readthedocs? (it's fine by me either way but wanted to know for future reference)